src/pages/item/index.js

```diff
@@ -454,13 +474,26 @@ function Item() {
                     </div>
                 </div>
                 {t('This item has not been observed on the Flea Market.')}
-                {t(" The maximum profitable price  is")}{' '}
+                {t(" The maximum profitable price is")}{' '}
                 {formatPrice(currentItemData.bestPrice)}
-                {t(', but the item may or may not sell at that price.')}{' '}
-                {t('The max profitable price is impacted by your intel center and hideout management skill levels in your settings.')}
+                {t(', but the item may not sell at that price.')}{' '}
+                {t('The max profitable price is impacted by the intel center and hideout management skill levels in your settings.')}
             </div>
         )
-        //fleaTooltip = t('No flea price seen');
+        if (currentItemData.cached) {
```

Comment:
Why are we using our own `cached` flang and not `isPlaceholderData` provided by `useQuery` when the data is _placeholder_?

Reply:
Because (1) I didn't know the placeholderData flag was a thing and (2) it's possible that when item data is being consumed, it may be removed from the context where the placeholder flag is available. 

I'm not sure which is the better way to do it. By adding the flag to each item, we can be sure it follows each individual item no matter where they're sent.

Reply:
Fair enough, we need to process items a lot for various parts. But then we also need to pay attention to copy the `cached `flag to transformed objects too.